mailer.rst

```diff
@@ -1204,9 +1204,40 @@ you have a transport called ``async``, you can route the message there:
                 ->senders(['async']);
         };
 
+Thanks to this, instead of being delivered immediately, messages will be sent
+to the transport to be handled later (see :ref:`messenger-worker`). Note that
+the "rendering" of the email (computed headers, body rendering, ...) is also
+deferred and will only happen just before the email is sent by the Messenger
+handler.
 
-Thanks to this, instead of being delivered immediately, messages will be sent to
-the transport to be handled later (see :ref:`messenger-worker`).
+.. versionadded:: 6.2
```

Comment:
Imho this should be a warning, otherwise it gets removed in Symfony 7 docs

Reply:
Having this removed in 7.0 is a good thing.

Reply:
Done then